test/e2e/runner.go

```diff
@@ -201,6 +201,23 @@ func main() {
 	}
 
 	if !cfg.SkipTest {
+		var resourceGroup string
+		if cliProvisioner.Account.ResourceGroup.Name != "" {
+			resourceGroup = cliProvisioner.Account.ResourceGroup.Name
+		} else {
+			resourceGroup = cfg.Name
+		}
+
+		if resourceGroup == "" {
+			if cfg.CleanUpIfFail {
+				teardown()
+			}
+			log.Fatalf("Resource Group is empty")
+			os.Exit(1)
```

Comment:
@jackfrancis can you confirm we're ok with an os.Exit() here?
(i generally avoid these calls at all costs)

Reply:
maybe returning an error would be cleaner?

Reply:
I found this from earlier in the code so I'll do whatever is recommended

Reply:
I removed it